server_addon/applications/server/settings.py

```diff
@@ -210,7 +220,14 @@ class ApplicationsAddonSettings(BaseSettingsModel):
         scope=["studio"]
     )
     only_available: bool = SettingsField(
-        True, title="Show only available applications")
+        True,
+        title="Show only available applications",
+        description="Enable to show available applications in AYON Launcher"
+                    " i.e. Show them if they are installed on the user"
+                    " machine. Note: AYON Launcher shows only the selected"
+                    " applications in project anatomy and this is another"
+                    " filter to ignore uninstalled apps."
```

Comment:
I feel like the _NOTE_ isn't that helpful. Maybe you like some parts of this re-write?
```suggestion
        description="Enable to show only available applications in AYON Launcher"
                    " for which the executable paths are found on the running machine."
                    " This applies as an additional filter to the applications defined in a "
                    " project's anatomy settings to ignore unavailable applications."
```